wagtail/admin/userbar.py

```diff
@@ -63,6 +63,36 @@ class AccessibilityItem(BaseItem):
     #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter-examples>`__.
     axe_rules = {}
 
+    #: A list used to add custom rules to the existing set of Axe rules, or to override the properties of existing Axe rules
+    #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure>`__.
+    axe_custom_rules = []
+
+    #: A custom rule to check the quality of the Images alt texts. Added to the list of custom rules and enabled by default.
+    #: Should be used in conjunction with :attr:`_axe_custom_alt_text_check`
+    #: This rule ensures that alt texts doesn't contain antipatterns like file extensions. Returns zero false positives.
+    _axe_custom_alt_text_rule = [
+        {
+            "id": "alt-text-quality",
+            "impact": "serious",
+            "selector": "img[alt]",
+            "tags": ["best-practice"],
+            "any": ["check-image-alt-text"],
+            "enabled": True,  # If ommited, defaults to True
```

Comment:
This should be set in the list of `axe_custom_rules` above rather than separate, so all rules can be modified in the same way.

Could we set `enabled` to `False` here, and enable it with `axe_run_only` like the other rules above?

Reply:
It seems that configurations in `spec` override whatever is configured in `runOnly`. It actually makes sense as the spec is used for overriding existing axe rules and checks. So I'm afraid, we'd need to treat custom rules separately.